CODEWALK.md

```diff
@@ -207,6 +221,13 @@ This block mining is spread through a few places in the code.
 Much in mining package, but also a bunch in the node implementation.
 - Chain protocol: protocol for exchange of mined blocks
 
+##### Protocol Mining APIs
+The [`storage`](https://github.com/filecoin-project/go-filecoin/tree/master/protocol/storage/),
+[`retrieval`](https://github.com/filecoin-project/go-filecoin/tree/master/protocol/retrieval/)
+and [`block`](https://github.com/filecoin-project/go-filecoin/tree/master/protocol/block/) packages now house their own APIs. These are the new interfaces for all mining commands, but not miner creation. These Protocol APIs provide a the new interface for the Network layer of go-filecoin.  Protocol APIs also consume Plumbing and Porcelain APIs. They are ephemeral, like the Porcelain API. Note also that the MiningOnce command uses `BlockMiningAPI` to create its own block mining worker, which lasts only for the time it takes to mine and post a new block.
```

Comment:
we should always refer to this as the 'block mining' protocol, as 'block' is so heavily overloaded